yaqd_wright/wright-continuous-filter-wheel.toml

```diff
@@ -10,8 +12,29 @@ bugtracker = "https://github.com/wright-group/yaqd-wright/issues"
 PyPI = "https://pypi.org/project/yaqd-wright"
 
 [config]
+motor.type = "int"
+motor.default = 1
+motor.doc = "Index for filter wheel motor" I'm not confident this is right...
+
+serial_port.type = "string"
+serial_port.default = "COM6" not actually sure about this...
```

Comment:
Should not have a default, defined by the `uses-uart` trait, so serial_port does not need to be included